Movie/Movie/DetailMoviePage/MoviesDescriptionViewController.swift

```diff
@@ -0,0 +1,353 @@
+            bookmarkButton.centerXAnchor.constraint(equalTo: bookmarkLabel.centerXAnchor, constant: 0),
+            bookmarkButton.bottomAnchor.constraint(equalTo: bookmarkLabel.topAnchor, constant: -5),
+            bookmarkButton.heightAnchor.constraint(equalToConstant: 20),
+            bookmarkButton.widthAnchor.constraint(equalToConstant: 20),
+            shareLabel.leadingAnchor.constraint(equalTo: bookmarkLabel.trailingAnchor, constant: 15),
+            shareLabel.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -5),
+            shareButton.centerXAnchor.constraint(equalTo: shareLabel.centerXAnchor, constant: 0),
+            shareButton.bottomAnchor.constraint(equalTo: shareLabel.topAnchor, constant: -5),
+            shareButton.heightAnchor.constraint(equalToConstant: 20),
+            shareButton.widthAnchor.constraint(equalToConstant: 20),
+            moreLabel.leadingAnchor.constraint(equalTo: shareLabel.trailingAnchor, constant: 15),
+            moreLabel.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -5),
+            moreButton.centerXAnchor.constraint(equalTo: moreLabel.centerXAnchor, constant: 0),
+            moreButton.bottomAnchor.constraint(equalTo: moreLabel.topAnchor, constant: -10),
+            moreButton.heightAnchor.constraint(equalToConstant: 7.5),
+            moreButton.widthAnchor.constraint(equalToConstant: 20),
+        ])
+        return view
+    }()
+
+    // MARK: - Private Property
+
+    private let dateFormater = DateFormatter()
+    private var actors: [Cast] = []
+    private var genres = String()
+
+    // MARK: Property
+
+    var data: Results?
+    var genre = String()
+
+    // MARK: - Lyfe Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupUI(data: data)
+        configureConstraint()
+    }
+
+    // MARK: Private Methods
+
+    private func setupUI(data: Results?) {
+        view.addSubview(moviePosterImageView)
+        view.addSubview(contentScrollView)
+        dateFormater.dateFormat = Constants.resultDateFormat
+        guard let dataPosterImage = data?.posterPath,
+              let dataRating = data?.voteAverage, let dataReleaseDate = data?.releaseDate, let movieId = data?.id
+        else { return }
+        guard let urlImage = URL(string: Constants.imageRequestURL + dataPosterImage) else { return }
+        let session = URLSession.shared
+        let task = session.dataTask(with: urlImage) { data, _, error in
+            guard let data = data, error == nil else { return }
+            DispatchQueue.main.async {
+                let image = UIImage(data: data)
+                self.moviePosterImageView.image = image
+            }
+        }
+        task.resume()
+
+        guard let url =
+            URL(
+                string: "https://api.themoviedb.org/3/movie/\(movieId)?api_key=d9e4494907230d135d6f6fd47beca82e&append_to_response=videos&language=ru"
```

Comment:
```suggestion
                string: ""
```
 все текстовки в константы